src/TMap.cpp

```diff
@@ -3096,6 +3096,9 @@ std::pair<bool, QString> TMap::readJsonMapFile(const QString& source, const bool
 
     mDefaultAreaName = mapObj[QLatin1String("defaultAreaName")].toString();
     mUnnamedAreaName = mapObj[QLatin1String("anonymousAreaName")].toString();
+    if (mapObj.contains(QLatin1String("userData"))){
```

Comment:
Another space is needed the other side as well...! :grinning: 
```suggestion
    if (mapObj.contains(QLatin1String("userData"))) {
```